reladomographql/src/main/java/com/gs/reladomo/graphql/FilterQueryBuilder.java

```diff
@@ -0,0 +1,386 @@
+/*
+ Copyright 2019 Goldman Sachs.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ */
+
+package com.gs.reladomo.graphql;
+
+import com.gs.fw.common.mithra.attribute.*;
+import com.gs.fw.common.mithra.finder.Operation;
+import com.gs.fw.common.mithra.finder.RelatedFinder;
+import com.gs.fw.common.mithra.util.MithraArrayTupleTupleSet;
+import org.eclipse.collections.impl.set.mutable.primitive.*;
+
+import java.sql.Timestamp;
+import java.util.*;
+import java.util.function.BiFunction;
+
+class FilterQueryBuilder
+{
+    private static final Map<String, BiFunction<StringAttribute, Object, Operation>> STRING_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<IntegerAttribute, Object, Operation>> INTEGER_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<LongAttribute, Object, Operation>> LONG_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<ShortAttribute, Object, Operation>> SHORT_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<DoubleAttribute, Object, Operation>> DOUBLE_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<FloatAttribute, Object, Operation>> FLOAT_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<TimestampAttribute, Object, Operation>> TIMESTAMP_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<AsOfAttribute, Object, Operation>> ASOF_OPS = new HashMap<> ();
+    private static final Map<String, BiFunction<BooleanAttribute, Object, Operation>> BOOLEAN_OPS = new HashMap<> ();
+
+    static
+    {
+        STRING_OPS.put ("eq", (attr, val) -> attr.eq ((String) val));

```

Comment:
I wonder if there is a better way to do this, like using reflection to find all public methods that return Operation. 

It also looks like you're missing two classes of method calls:
1) Methods that operate on the attribute and return another attribute. Examples include `subString`, `toLowerCase`, etc for StringAttribute. `abs` for numerical attributes and so on.
2) Methods that combine attributes together, such as the math operations in numerical attributes. Query's that look like `a().plus(b()).greaterThan(10)` can't be represented otherwise.